src/app/analyze/stack/stack-analyses.service.ts

```diff
@@ -8,10 +9,10 @@ import 'rxjs/add/operator/map';
 @Injectable()
 export class StackAnalysesService {
 
-  readonly STACK_ANALYSES_BASE_URL: string
-    = 'http://ose-vm1.lab.eng.blr.redhat.com:32000/api/v1/stack-analyses/';
+  private stackAnalysesUrl = process.env.STACK_API_URL;
```

Comment:
That's not plural (or even a noun). The term "analyses" in any correct form of use is a verb with third-person singular conjugation (as in, "He analyses this").

I understand it's not a biggie, but the entire point of having verbose variable names is for them to be humnaly understandable; otherwise `a`, `b`, `c` etc. are perfectly viable.